researchspace/researchspace
# Add custom wrapper for QLever sparql repository.

# Why

QLever - https://github.com/ad-freiburg/qlever doesn't support ASK queries, so we rewrite ASK queries to `SELECT * {} LIMIT 1`. This PR reuse similar wrapper from existing Virtuoso wrapper.

# What

Create a new RDF4J repository for query rewriting.

# How To Test

1. start a sample qlever instance by using https://github.com/ad-freiburg/qlever-control/ 
2. create new platform repository from the UI using "qlever" template from the dropdown list
3. try to send ASK query from sparql endpoint UI, it should work
